SwiftPackageManager.md

```diff
@@ -51,6 +51,10 @@ https://github.com/firebase/firebase-ios-sdk/issues/6472#issuecomment-694449182.
 
 <img src="docs/resources/SPMObjC.png">
 
+If you're using FirebaseCrashlytics, use
+`${BUILD_DIR%Build/*}SourcePackages/checkouts/firebase-ios-sdk/Crashlytics/run`
```

Comment:
Does `%Build` get populated by Xcode for the run script?

Reply:
Good point, it does not and we only need the BUILD_DIR part.